tests/test_process.py

```diff
@@ -15,6 +15,7 @@
 import pytest
 
 from hades import Event, Hades, PredefinedEventAdder, Process, ProcessUnregistered, RandomProcess, SimulationStarted
+from hades.core.event import SimulationEnded
```

Comment:
may be worth adding to relevant __init__.py so it can be imported like `SimulationStarted`